docs/arch/adr-003-versioning-branching-strategy.md

```diff
@@ -52,6 +56,18 @@ Based off of the release version and expectations, the version type's branching
 
  ![Branching Strategy](images/ReleasingBranchStrategy.png)
 
+### Changes for Specific Release Versions
+There will be times that we will want to target changes to go into specific release versions. Below outlines 2 different scenarios in which this can occur and how we will approach them.
+
+1. If a release branch already exists: This is the most common scenario where we want to add a change to an upcoming release and the release branch has already been cut. In this instance, developers will commit their changes to `main` first. Those changes (or some form of them) will then be applied to the release branch in question so that they can be included in the specific release version. 
+\* Disclaimer: the how and when to apply changes to a release branch will be covered in another upcoming ADR.
+
+1. If a release branch is not yet cut: This is the scenario where we start to develop a feature for a future version before we have cut a release branch for the upcoming version (eg. we want to work on a feature for `1.2.0` but we haven't cut the release branch for `1.1.0` yet). 
```

Comment:
all the number bullets are "1"

Reply:
In markdown, you just use `1.` everywhere and when markdown renders, it uses the correct numbers. If you look at this in the rich diff view for the PR, it shows correctly for me. Let me know if that isn't the case for you